src/PowerShellEditorServices/Server/PsesDebugServer.cs

```diff
@@ -21,7 +22,10 @@ namespace Microsoft.PowerShell.EditorServices.Server
     /// </summary>
     internal class PsesDebugServer : IDisposable
     {
-        private static bool s_hasRunPsrlStaticCtor = false;
+        /// <summary>
+        /// This is a bool but must be an int, since Interlocked.Exchange can't handle a bool
+        /// </summary>
+        private static int s_hasRunPsrlStaticCtor = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this initialization to 's_hasRunPsrlStaticCtor', the compiler will do that for you.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4785996)